roles/idrac_os_deployment/templates/UBUNTU_jammy.j2

```diff
@@ -0,0 +1,43 @@
+#cloud-config
+# vim: ft=yaml:
+autoinstall:
+  version: 1
+  early-commands:
+    - ["cat", "/autoinstall.yaml"]
+  refresh-installer:
+    update: yes
+  apt:
+    preserve_sources_list: false
+    mirror-selection:
+      primary:
+        - country-mirror
+        - uri: "http://archive.ubuntu.com/ubuntu"
+          arches: [i386, amd64]
+        - uri: "http://ports.ubuntu.com/ubuntu-ports"
+          arches: [s390x, arm64, armhf, powerpc, ppc64el, riscv64]
```

Comment:
Good to move these to a vars as well.  In a Data center sometimes they would not have access to the internet and want to use there own mirror.

Reply:
have moved them to vars